Final_Project/Main.h

```diff
@@ -9,6 +9,8 @@ static char Name[] = "GTID";
 static DWORD MAX_RANDOM_WAIT_TIME_MS = 500;
 static DWORD IDLE_TIMEOUT_TIME_S = 30;
 static DWORD CHECK_IDLE_TIMEOUT_MS = 5000;
+LPCSTR lpszCommName = "com1";
+char str[80] = "";
```

Comment:
Correct me if I'm wrong, but I think only static values should go in the header file. Being that `lpszCommName` should never change I think we can make that static, but I think `str` should probably go back in the main file.

Reply:
Will change, I was kind of shooting in the dark on where to put them